internal/kgateway/proxy_syncer/proxy_syncer.go

```
@@ -179,7 +181,7 @@ func (r report) Equals(in report) bool {
 	return true
 }
 
-func (s *ProxySyncer) Init(ctx context.Context, krtopts krtutil.KrtOptions) error {
+func (s *ProxySyncer) Init(ctx context.Context, krtopts krtutil.KrtOptions) {
```

is there no case where a proxy syncer should throw an error?